lib/PlaybackArea.tsx

```diff
@@ -157,16 +153,17 @@ const search = (api: string, parameters: VapixParameters = {}) => {
 
 export const PlaybackArea: React.FC<PlaybackAreaProps> = ({
   forwardedRef,
-  host,
-  api,
-  parameters = {},
-  play,
-  refresh,
   onPlaying,
   onSdp,
   metadataHandler,
   secure = window.location.protocol === 'https',
 }) => {
+  const { host, api, parameters, refresh } = useContext(VideoPlayerContext)
+
+  if (api === undefined) {
```

Comment:
So, this one stumped me. For _no_ apparent reason, `api` was `undefined` in `Player.tsx` but worked just fine in `BasicPlayer.tsx`. I'd love for someone to help me find out why that is. I tried for 30 minutes before I thought "f it" and implemented this guard.

Reply:
If you rebase this on the latest commit, `api` is gone (and replaced by a stricter typed `format`). Maybe that will solve the issue?